tests/codex/blockexchange/discovery/testdiscovery.nim

```diff
@@ -5,7 +5,7 @@ import std/tables
 import pkg/asynctest
 import pkg/chronos
 
-import pkg/libp2p
+import pkg/libp2p except setup
```

Comment:
in order to avoid doing `except setup`, we might want to create a top level re-export (in the test directory) with

```nim
import pkg/libp2p

export libp2p except setup
```

Reply:
Yah it crossed my mind too, after updating 20 of them. :) It'd be nice just to have it. We do already have a `helper.nim` test thing, but maybe we'd want a new one? 

I'll let it stew a bit over the weekend and see on Monday.

Reply:
Yea, probably a dedicated `lphelper` or something like that.

Reply:
I just stuck it in `tests/helper.nim` since it's already used in all of the unit tests that used libp2p.